include/experimental/__p0009_bits/utility.hpp

```diff
@@ -64,6 +66,112 @@ constexpr struct
   }
 } stride;
 
+template<class T>
+MDSPAN_INLINE_FUNCTION
+constexpr void maybe_unused_variable(const T&) {}
+
+
+// same as std::integral_constant but with __host__ __device__ annotations on
+// the implicit conversion function and the call operator
```

Comment:
Why do we need/want conversion from to the std:: counterpart?

Reply:
because there are places where we pass in std::integral_constant, and std::integral_constant is the standard approved way to pass certain ways to certain functions. For example `submdspan(a, std::integral_constant<int, 1>())` is fine in device code.